README.md

```diff
@@ -82,16 +85,32 @@ Write the BNF (not EBNF) description for this language.
 
 ## A2.1
 
-«replace this with your answer»
+<sentence> ::= "the" <adjectives> <noun> <verb> <adverb>
 
+<adjectives> ::= "" | <adjective> | <adjective> <adjective>
```

Comment:
This allows for 1 or 2 adjectives, but the original had them just repeating. That would be

~~~
<adjectives> ::= "" | <adjective> | <adjectives> <adjective>
~~~

# -2